src/foam/nanos/crunch/Capability.js

```diff
@@ -356,24 +348,29 @@ foam.CLASS({
           subject.setUser((User) userDAO.find(ucj.getSourceId()));
         }
 
-        for ( CapabilityCapabilityJunction ccJunction : ccJunctions ) {
-          Capability cap = (Capability) ccJunction.findSourceId(x);
-          if ( ! cap.getEnabled() ) continue;
-          UserCapabilityJunction ucJunction = crunchService.getJunctionForSubject(x, ccJunction.getTargetId(), subject);
-
-          if ( ucJunction != null && ucJunction.getStatus() == CapabilityJunctionStatus.GRANTED )
-            continue;
-
-          if ( ucJunction == null ) {
-            return CapabilityJunctionStatus.ACTION_REQUIRED;
-          }
-          if ( ucJunction.getStatus() != CapabilityJunctionStatus.GRANTED
-               && ucJunction.getStatus() != CapabilityJunctionStatus.PENDING ) {
-            return CapabilityJunctionStatus.ACTION_REQUIRED;
+        var prereqs = crunchService.getPrereqs(getId());
+        if ( prereqs != null ) {
+          for ( var capId : prereqs ) {
+            // QUESTION if the capability is disabled what would be the status of the prereqs chain?
```

Comment:
Just ignore it. Maybe in the future we should prevent users from disabling capabilities which are prerequisits.

Reply:
so `cap.getPrereqsChainedStatus(x, ucj)` would return GRANTED if cap is disabled.

Reply:
yes

Reply:
Fixed in https://github.com/foam-framework/foam2/pull/4010/commits/ec0b01c80288c3f553b265b7aac531120c918838.